src/Entities/Instalment.php

```diff
@@ -47,4 +47,7 @@ class Instalment extends Base
 
     /** @var int */
     public $customer_fee;
+
+	/** @var int */
+    public $customer_interest;
```

Comment:
We might want to add this after that: https://github.com/alma/main/pull/5055

```suggestion

	/** @var int */
    public $customer_interest;

	/** @var int */
    public $total_amount;
```